system-parachains/asset-hubs/asset-hub-kusama/src/genesis_config_presets.rs

```diff
@@ -69,7 +69,7 @@ fn asset_hub_kusama_genesis(
 		},
 		"staking": {
 			"validatorCount": 1000,
-			"devStakers": Some((2_000, 25_000)),
+			"devStakers": Some((4_000, 15_000)),
```

Comment:
I had some TODO in a checklist for you to set some Genesis config values for Kusama, is this it? Can i Check that off?

Reply:
Yeah this is correct now and no benchmarks shoulf fail.